awesomplete.js

```diff
@@ -265,6 +274,10 @@ _.prototype = {
 		else {
 			this.close();
 		}
+	},
+
+	setStatus: function(message) {
+		this.status.innerHTML = message;
```

Comment:
I agree on using `textContent` for both. I think for now it's ok to also get rid of the setStatus helper Method as there is only one place that sends status messages to the live-region. I we decide to maybe add an info status that announces that search results have been found and how many and that the user can use the cursor keys to find them then adding it back would make sense.